x-pack/plugins/monitoring/public/components/cluster/listing/listing.js

```diff
@@ -132,7 +132,7 @@ const getColumns = (
       'data-test-subj': 'nodesCount',
       sortable: true,
       render: (total, cluster) => (
-        <IsClusterSupported {...cluster}>{numeral(total).format('0,0')}</IsClusterSupported>
+        <IsClusterSupported {...cluster}>{numeral(total || 0).format('0,0')}</IsClusterSupported>
```

Comment:
I think this could work, but it's sort of lying to you. I was thinking it might be better to swap in an "N/A" instead of inventing the zero.